index.bs

```diff
@@ -181,12 +182,18 @@ The <a href="https://www.w3.org/TR/webxr/">WebXR Device API API</a> defines each
 
 Augmented Reality devices that use transparent optical elements frequently exhibit {{XREnvironmentBlendMode/"additive"}} blending behavior even for {{XRSessionMode/"immersive-vr"}} sessions.
 
-The {{XRSession/environmentBlendMode}} MUST NOT be {{opaque}} for {{immersive-ar}} sessions. When an [=xr device=] [=performs automatic composition=] using a passthrough camera in an {{XRSessionMode/"immersive-ar"}} session, it MUST report {{XREnvironmentBlendMode/"alpha-blend"}} blending behavior.
+The {{XRSession/environmentBlendMode}} MUST NOT be {{opaque}} for {{immersive-ar}} sessions. When an [=xr device=] [=perform automatic composition|performs automatic composition=] using a passthrough camera in an {{XRSessionMode/"immersive-ar"}} session, it MUST report {{XREnvironmentBlendMode/"alpha-blend"}} blending behavior.
 
 Automatic composition {#automatic-composition}
 ---------
-<dfn>performs automatic composition</dfn>
-TODO add explanation here.
+
+An [=xr device=] will have its [=xr compositor=] <dfn>perform automatic composition</dfn> by drawing the rendered scene on top of the real-world background. Such a process MUST NOT grant the page access to any information about the real world, like camera data or geometry.
```

Comment:
> Such a process MUST NOT grant the page access to any information about the real world, like camera data or geometry.

How about 

> Such a process MUST NOT grant the page access to any additional information such as camera intrinsics, media streams, real-world geometry, etc.

Reply:
Also, I think it might be useful to move this sentence a bit further down after the explanations in the next paragraph.

Reply:
Maybe add "unless explicitly allowed by the user" and/or "MUST not automatically".
Current wording implies that no camera or geometry is ever allowed.

Reply:
Added the "automatically", the intent was that _automatic composition_ doesn't grant access, something else might.